### PR TITLE
History view

### PR DESCRIPTION
**Pull request by tzelic**

### CODE REVIEW

File: server/db/seeds.js [hunk 1]
```diff
@@ -50,8 +50,11 @@ function generateOodlets(quantity, oodlers, quantifiedThingies) {
   console.log(`Generating ${quantity} Oodlets...`);
 
   for (let i = 0; i < quantity; i++) {
+    let randCreateDate = faker.date.past();
```
Comment (b0rza) [inline server/db/seeds.js]: How about this:
`let notSoRandCreateDate = moment().subtract(2*quantity, 'weeks').toDate()`

That way we'll get a nice simulation of a real Oodlet backlog

File: server/routes/oodlet.js [hunk 2]
```diff
@@ -4,14 +4,14 @@
 'use strict';
 
 const Oodlet = require('../models/oodlet');
+const moment = require('moment');
 
 function list(request, reply) {
-  let limit = request.query.limit || 5;
-  let offset = request.query.offset || 0;
+  let fromDate = moment(request.query.fromDate).toDate() || moment().subtract(3, 'months').toDate();
```
Comment (b0rza) [inline server/routes/oodlet.js]: :+1:

File: server/db/seeds.js [hunk 3]
```diff
@@ -39,7 +39,7 @@ function generateQuantifiedThingies(quantity, callback) {
     })
     .save()
     .then((thingy) => {
-      quantifiedThingies.push(object.merge(thingy, { qty: faker.random.number(10) }));
+      quantifiedThingies.push(object.merge(thingy, { qty: faker.random.number({ 'min': 1,'max': 30 })}));
```
Comment (b0rza) [inline server/db/seeds.js]: GG!

File: client/views/HistoryView.vue [hunk 4]
```diff
@@ -13,20 +18,34 @@
...
+      load(){
+        this.$http.get('/oodlet', {
+          params: {
+            fromDate: moment(this.fromDate).format(),
+            toDate: moment(this.toDate).format()
+          }})
+          .then((response) => {
+            this.oodlets = response.body;
+          },
+          (err)=>console.log('Error!!'));
```
Comment (b0rza) [inline client/views/HistoryView.vue]: WHY U NO WHITESPACE

File: client/views/HistoryView.vue [hunk 5]
```diff
@@ -13,20 +18,34 @@
...
+    },
+
+    methods: {
+      load(){
+        this.$http.get('/oodlet', {
+          params: {
+            fromDate: moment(this.fromDate).format(),
+            toDate: moment(this.toDate).format()
+          }})
+          .then((response) => {
```
Comment (b0rza) [inline client/views/HistoryView.vue]: Don't use parenthesis for arrow function if only a single parameter.

File: server/routes/index.js [hunk 6]
```diff
@@ -3,6 +3,7 @@ const boom = require('boom');
 function errorHandler(defaultHandler) {
   return function(request, reply) {
     return defaultHandler(request, reply).catch((err) => {
+      console.log(err);
```
Comment (b0rza) [inline server/routes/index.js]: :+1:

**Comment by b0rza** [comment]
Small stuff and lets merge ;)

File: client/views/HistoryView.vue (hunk 4 above)
Comment (tzelic) [inline client/views/HistoryView.vue]: :disappointed:

File: client/views/HistoryView.vue (hunk 5 above)
Comment (tzelic) [inline client/views/HistoryView.vue]: :+1:

File: server/db/seeds.js (hunk 1 above)
Comment (tzelic) [inline server/db/seeds.js]: Nice! :100:

Comment (tzelic) [inline server/db/seeds.js]: you mean 
let notSoRandCreateDate = moment().subtract(2 * i, 'weeks').toDate()
2*quantity is always same value :smile:

Comment (b0rza) [inline server/db/seeds.js]: GOTCHA! :P

File: server/db/seeds.js [hunk 7]
```diff
@@ -50,11 +51,12 @@ function generateOodlets(quantity, oodlers, quantifiedThingies) {
...
-    let randCreateDate = faker.date.past();
+    let createDate = moment().subtract(2*i, 'weeks').toDate();
+
     Oodlet({
-      createdAt: randCreateDate,
-      updatedAt: randCreateDate,
-      dueDate: new Date(randCreateDate.getTime()+1000*60*60*24*14),
+      createdAt: createDate,
+      updatedAt: createDate,
+      dueDate: new Date(createDate.getTime()+1000*60*60*24*14),
```
Comment (b0rza) [inline server/db/seeds.js]: Use moment so we don't have to use a DIGITRON :stuck_out_tongue_closed_eyes:

**Pull request merged**